src/ds/index.js

```diff
@@ -78,6 +78,8 @@ $(document).ready(function() {
         $("#choosetools").toggleClass("d-none");
         $("#done_button").toggleClass("d-none").toggleClass("display-block");
         $("#savedchoices").removeClass('choose').addClass('edit');
+        $("h1.choose").text('Edit institutions');
```

Comment:
@leifj I was going to make two different HTML elements in the index.html file, but this seems a bit more elegant solution. What I'm not certain of is: would you prefer to change text only in the HTML files and use JS to only switch classes, or is this fine?